blog/try-it-yourself.md

```diff
@@ -121,7 +121,7 @@ After you finish all the steps above, you can start the analytical process.
 If you want to know the table schema, you can use `show create table tbl_name` to get that information.
 :::
 
-Because you have imported the sample data of all GitHub events occurred on January 1, 2022, you can start to make any queries based on that data by using SQL commands. 
+Because you have imported the sample data of all GitHub events occurred on the first hour of 2022 (from 2022-01-01 00:00:00 to 2022-01-01 00:59:59), you can start to make any queries based on that data by using SQL commands. 
```

Comment:
```suggestion
Because you have imported the sample data of all GitHub events occurred in the first hour of the first day of 2022 (from 2022-01-01 00:00:00 to 2022-01-01 00:59:59), you can start to make any queries based on that data by using SQL commands. 
```